08:55:54.456[DBAgent-4182][ERROR] 106. PreparedStatement.executeBatch() FAILED! batching 12 statements:
 1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267876
           ,'01'
           ,3
           ,'TMT22d38a2d3a24434e947c8f34d5533253192413967434680590'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267880
           ,'01'
           ,3
           ,'TMTf15d0928b82d45e7a37ef8a3b65b9f6f192413967507111560'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 3:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267882
           ,'01'
           ,3
           ,'TMTc9cfbaa77e4e4101b555604a84e269f4192413967568104710'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 4:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267886
           ,'01'
           ,3
           ,'TMT715a22773f6c48cd9382920739a61985192413967649971720'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 5:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267889
           ,'01'
           ,3
           ,'TMT20150840cbe24174bda931dcacb0131c192413967717884410'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 6:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267891
           ,'01'
           ,3
           ,'TMTc69b3f8b510d433691321582efe3f5be192413967790591540'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 7:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267897
           ,'01'
           ,3
           ,'TMT5b39f30e4eb14f08b9d58397f994744d192413967859687690'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 8:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267899
           ,'01'
           ,3
           ,'TMT86024ec3e3ab4356b90b4ea1cabce60d192413967927578520'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 9:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267903
           ,'01'
           ,3
           ,'TMT4b1ae040a2a94fc4ab505bcde7901aeb192413967993640890'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
10:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267906
           ,'01'
           ,3
           ,'TMT6c41a49902304f5ea9dae742a54eaed3192413968055561930'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
11:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267910
           ,'01'
           ,3
           ,'TMTf34091ac5dac422eac1d553a1386ce27192413968122550790'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
12:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267914
           ,'01'
           ,3
           ,'TMTccccfb77ef64435aae80132fe6a1c1c9192413968184199870'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 3 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5267914-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267914-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
08:55:54.458[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267876
           ,'01'
           ,3
           ,'TMT22d38a2d3a24434e947c8f34d5533253192413967434680590'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267876-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.459[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267880
           ,'01'
           ,3
           ,'TMTf15d0928b82d45e7a37ef8a3b65b9f6f192413967507111560'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267880-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.460[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267882
           ,'01'
           ,3
           ,'TMTc9cfbaa77e4e4101b555604a84e269f4192413967568104710'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267882-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.461[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267886
           ,'01'
           ,3
           ,'TMT715a22773f6c48cd9382920739a61985192413967649971720'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267886-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.462[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267889
           ,'01'
           ,3
           ,'TMT20150840cbe24174bda931dcacb0131c192413967717884410'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267889-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.463[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267891
           ,'01'
           ,3
           ,'TMTc69b3f8b510d433691321582efe3f5be192413967790591540'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267891-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.465[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267897
           ,'01'
           ,3
           ,'TMT5b39f30e4eb14f08b9d58397f994744d192413967859687690'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267897-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.466[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267899
           ,'01'
           ,3
           ,'TMT86024ec3e3ab4356b90b4ea1cabce60d192413967927578520'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267899-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.467[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267903
           ,'01'
           ,3
           ,'TMT4b1ae040a2a94fc4ab505bcde7901aeb192413967993640890'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267903-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.468[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267906
           ,'01'
           ,3
           ,'TMT6c41a49902304f5ea9dae742a54eaed3192413968055561930'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267906-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.470[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267910
           ,'01'
           ,3
           ,'TMTf34091ac5dac422eac1d553a1386ce27192413968122550790'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267910-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
08:55:54.471[DBAgent-4182][ERROR] 106. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5267914
           ,'01'
           ,3
           ,'TMTccccfb77ef64435aae80132fe6a1c1c9192413968184199870'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5267914-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:14:01.453[DBAgent-4355][ERROR] 44. PreparedStatement.executeBatch() FAILED! batching 3 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381612
           ,'01'
           ,3
           ,'TMT9d3b9de09a9949928bf24920ad00d4e8192604741116360180'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381615
           ,'01'
           ,3
           ,'TMT00f9741c33364b48a63ca9415e521839192604750630437110'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
3:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381618
           ,'01'
           ,3
           ,'TMTca6f65ed178f4bc3be1c4d65185d03a2192604769879433050'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5381618-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5381618-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
14:14:01.454[DBAgent-4355][ERROR] 44. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381612
           ,'01'
           ,3
           ,'TMT9d3b9de09a9949928bf24920ad00d4e8192604741116360180'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5381612-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:14:01.456[DBAgent-4355][ERROR] 44. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381615
           ,'01'
           ,3
           ,'TMT00f9741c33364b48a63ca9415e521839192604750630437110'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5381615-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:14:01.457[DBAgent-4355][ERROR] 44. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5381618
           ,'01'
           ,3
           ,'TMTca6f65ed178f4bc3be1c4d65185d03a2192604769879433050'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5381618-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:22:01.453[DBAgent-4360][ERROR] 96. PreparedStatement.executeBatch() FAILED! batching 2 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384570
           ,'01'
           ,3
           ,'TMTa078f7a04e61423f9fc91b8949911477192609217113083110'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384722
           ,'01'
           ,3
           ,'TMT7d3adb3097804b37878b726a1e1294e9192609421083549290'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5384722-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384722-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
14:22:01.458[DBAgent-4360][ERROR] 96. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384722
           ,'01'
           ,3
           ,'TMT7d3adb3097804b37878b726a1e1294e9192609421083549290'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384722-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:23:03.454[DBAgent-4351][ERROR] 114. PreparedStatement.executeBatch() FAILED! batching 6 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5385510
           ,'01'
           ,3
           ,'TMTdf8106e40d7047f58af037c7770a94d0192610300912275490'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5385506
           ,'01'
           ,3
           ,'TMT184ef2b6a2b04dc3a3b9ed269f5b31f6192610300910880680'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
3:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5385508
           ,'01'
           ,3
           ,'TMTd085e1a829e04b8397bfc6cc4ad4026d192610300911854340'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
4:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384776
           ,'01'
           ,3
           ,'TMT854b7f1e6d1c4664835dda77cd6e32cf192609523716156930'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
5:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384826
           ,'01'
           ,3
           ,'TMTca49db419d83476a90dbfdfdf457851e192609605696680050'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
6:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384996
           ,'01'
           ,3
           ,'TMT2fab019eacaf43478d07b3682abb918e192609839801173140'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 2 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5384996-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384996-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
14:23:03.460[DBAgent-4351][ERROR] 114. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384776
           ,'01'
           ,3
           ,'TMT854b7f1e6d1c4664835dda77cd6e32cf192609523716156930'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384776-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:23:03.461[DBAgent-4351][ERROR] 114. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384826
           ,'01'
           ,3
           ,'TMTca49db419d83476a90dbfdfdf457851e192609605696680050'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384826-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:23:03.462[DBAgent-4351][ERROR] 114. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5384996
           ,'01'
           ,3
           ,'TMT2fab019eacaf43478d07b3682abb918e192609839801173140'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5384996-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
14:27:19.452[DBAgent-4366][ERROR] 116. PreparedStatement.executeBatch() FAILED! batching 1 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5387977
           ,'01'
           ,3
           ,'TMT348348e79ed04d7fbb61ef708995ecee192612331892090760'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5387977-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5387977-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
14:27:19.454[DBAgent-4366][ERROR] 116. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5387977
           ,'01'
           ,3
           ,'TMT348348e79ed04d7fbb61ef708995ecee192612331892090760'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5387977-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
15:19:02.453[DBAgent-4329][ERROR] 116. PreparedStatement.executeBatch() FAILED! batching 2 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5410467
           ,'01'
           ,3
           ,'TMTc26d64ec73124ac187059572cdbb982a192643743362145710'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5410470
           ,'01'
           ,3
           ,'TMT3ef7be4887b94ae8850edf3d9d0d02af192643749651853060'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5410470-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5410470-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
15:19:02.454[DBAgent-4329][ERROR] 116. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5410467
           ,'01'
           ,3
           ,'TMTc26d64ec73124ac187059572cdbb982a192643743362145710'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5410467-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
15:19:02.455[DBAgent-4329][ERROR] 116. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5410470
           ,'01'
           ,3
           ,'TMT3ef7be4887b94ae8850edf3d9d0d02af192643749651853060'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5410470-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
16:04:16.453[DBAgent-4410][ERROR] 60. PreparedStatement.executeBatch() FAILED! batching 1 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5432623
           ,'01'
           ,3
           ,'TMT7ddade167e8445958c7ca903f4f95366192670978076809460'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5432623-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5432623-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
16:04:16.454[DBAgent-4410][ERROR] 60. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5432623
           ,'01'
           ,3
           ,'TMT7ddade167e8445958c7ca903f4f95366192670978076809460'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5432623-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
16:34:26.453[DBAgent-4413][ERROR] 17. PreparedStatement.executeBatch() FAILED! batching 1 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5448541
           ,'01'
           ,3
           ,'TMT8dc95a17c7a04e17bcb75c28bea1eee0192688987534220960'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5448541-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5448541-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
16:34:26.454[DBAgent-4413][ERROR] 17. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5448541
           ,'01'
           ,3
           ,'TMT8dc95a17c7a04e17bcb75c28bea1eee0192688987534220960'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5448541-01' for key 'PRIMARY'
	at sun.reflect.GeneratedConstructorAccessor127.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
18:08:18.453[DBAgent-4440][ERROR] 5. PreparedStatement.executeBatch() FAILED! batching 3 statements:
1:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506078
           ,'01'
           ,3
           ,'TMT40cdbeafd9cf403d8f6f269d70b840f5192745351914532780'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
2:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506130
           ,'01'
           ,3
           ,'TMT0e7072fa2cd04e1fb298f5f426c43857192745423706297180'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
3:  INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506138
           ,'01'
           ,3
           ,'TMTad97dab3db6a4478a4e1dd359271c308192745424893548620'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
java.sql.BatchUpdateException: Duplicate entry '3-5506138-01' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createBatchUpdateException(SQLError.java:1168)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1823)
	at com.mysql.jdbc.PreparedStatement.executeBatchInternal(PreparedStatement.java:1307)
	at com.mysql.jdbc.StatementImpl.executeBatch(StatementImpl.java:970)
	at net.sf.log4jdbc.sql.jdbcapi.StatementSpy.executeBatch(StatementSpy.java:571)
	at sun.reflect.GeneratedMethodAccessor105.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.executeBatch(Unknown Source)
	at org.apache.ibatis.executor.BatchExecutor.doFlushStatements(BatchExecutor.java:122)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:129)
	at org.apache.ibatis.executor.BaseExecutor.flushStatements(BaseExecutor.java:122)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.flushStatements(DefaultSqlSession.java:253)
	at sun.reflect.GeneratedMethodAccessor104.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.flushStatements(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.flushStatements(SqlSessionTemplate.java:394)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItems(AsyncDatabaseExecutor.java:991)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$FastClassBySpringCGLIB$$8dd33f65.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.AsyncDatabaseExecutor$$EnhancerBySpringCGLIB$$ed8ea605.insertPushDeviceSuccessPpcStatusWithPushItems(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:328)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5506138-01' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
	at com.mysql.jdbc.PreparedStatement.executeBatchSerially(PreparedStatement.java:1801)
	... 36 common frames omitted
18:08:18.454[DBAgent-4440][ERROR] 5. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506078
           ,'01'
           ,3
           ,'TMT40cdbeafd9cf403d8f6f269d70b840f5192745351914532780'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 0 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5506078-01' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
18:08:18.456[DBAgent-4440][ERROR] 5. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506130
           ,'01'
           ,3
           ,'TMT0e7072fa2cd04e1fb298f5f426c43857192745423706297180'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5506130-01' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
18:08:18.457[DBAgent-4440][ERROR] 5. PreparedStatement.execute() FAILED! INSERT INTO MPPS_PUSH_DEVICE_SUCCESS
           (
           PUSH_DEVICE_ID
           ,PUSH_TYPE
           ,SYSTEM_ID
           ,DEVICE_MSG_UUID
           ,STATUS
           ,FAIL_CNT
           ,REG_DT
           ,UPT_DT
           )
     VALUES
           (
           5506138
           ,'01'
           ,3
           ,'TMTad97dab3db6a4478a4e1dd359271c308192745424893548620'
           ,0
           ,0
           ,NOW(3)
           ,NOW(3)
           )
 {FAILED after 1 msec}
com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry '3-5506138-01' for key 'PRIMARY'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1242)
	at net.sf.log4jdbc.sql.jdbcapi.PreparedStatementSpy.execute(PreparedStatementSpy.java:443)
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:46)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:198)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:185)
	at sun.reflect.GeneratedMethodAccessor115.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:433)
	at com.sun.proxy.$Proxy78.insert(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.insert(SqlSessionTemplate.java:278)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:58)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:59)
	at com.sun.proxy.$Proxy79.insertPrivatePushSuccessStatus(Unknown Source)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor.insertPushDeviceSuccessPpcStatusWithPushItem(SyncDatabaseExecutor.java:425)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$FastClassBySpringCGLIB$$464c476e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:736)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:671)
	at com.happyhouse.servers.ps.service.SyncDatabaseExecutor$$EnhancerBySpringCGLIB$$37568bc0.insertPushDeviceSuccessPpcStatusWithPushItem(<generated>)
	at com.happyhouse.servers.ps.service.DatabaseMediator.insertPushDeviceSuccessPpcStatusWithPushItems(DatabaseMediator.java:333)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1039)
	at com.happyhouse.servers.ps.scheduler.DbAgentMessageScheduler$DBAgentPushDeviceSuccessPpcStatusForUpdate.call(DbAgentMessageScheduler.java:1020)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
